lib/learnosity/sdk/request/init.rb

```diff
@@ -3,226 +3,271 @@
 
 require 'learnosity/sdk/exceptions'
 require 'learnosity/sdk/utils'
+require 'learnosity/sdk/version'
+require 'sys/uname'
 
 module Learnosity
   module Sdk
     module Request
 
       class Init
-	# XXX: Needs to be public for unit tests
-	attr_reader :security_packet, :request_string
+        # XXX: Needs to be public for unit tests
+        attr_reader :security_packet, :request_string
 
-        # Keynames that are valid in the security_packet, they are also in
-        # the correct order for signature generation.
-        @@validSecurityKeys = ['consumer_key', 'domain', 'timestamp', 'expires', 'user_id'];
+        class <<self
+          attr_accessor :telemetry_enabled
```

Comment:
In our other SDKs we have made this variable private and required the use of methods to do all operations. The `attr_accessor` sets up a getter and setter, making it possible to change `telemetry_enabled` without using the methods below.

For accessing this this variable, we should probably just add an extra class method called `telemetry_enabled?` instead of providing this accessor.